src/data/bucket/symbol_bucket.js

```diff
@@ -61,17 +60,18 @@ const symbolInterfaces = {
         ]
     },
     collisionBox: { // used to render collision boxes for debugging purposes
-        layoutVertexArrayType: createVertexArrayType([
+        layoutAttributes: [
             {name: 'a_pos',     components: 2, type: 'Int16'},
             {name: 'a_extrude', components: 2, type: 'Int16'},
             {name: 'a_data',    components: 2, type: 'Uint8'}
-        ]),
+        ],
         elementArrayType: createElementArrayType(2)
     }
 };
 
-function addVertex(array, x, y, ox, oy, tx, ty, minzoom, maxzoom, labelminzoom, labelangle) {
-    array.emplaceBack(
+function addVertex(array, x, y, ox, oy, tx, ty, sizeData, minzoom, maxzoom, labelminzoom, labelangle) {
+    array.emplaceBack.apply(array,
```

Comment:
Further profiling confirms that `apply` is a bad idea here.

Timing `emplaceBack()` (`[min, 1st quartile, median, 3rd quartile, max]`):

 - master (i.e. w/o the size data added in this PR): `[ 0.001, 0.004, 0.005, 0.006, 0.778 ]`
 - using apply: `[ 0.002, 0.008, 0.01, 0.011, 12.945 ]`

Changed to avoid using apply by always passing three (possibly undefined) args for the size data; timing for emplaceBack with that approach is basically identical to master.